frontend/packages/knative-plugin/src/components/add/event-sources/PingSourceSection.tsx

```diff
@@ -5,7 +5,12 @@ import FormSection from '@console/dev-console/src/components/import/section/Form
 
 const PingSourceSection: React.FC = () => (
   <FormSection title="PingSource" extraMargin>
-    <InputField type={TextInputTypes.text} name="data.pingsource.data" label="Data" />
+    <InputField
+      type={TextInputTypes.text}
+      name="data.pingsource.data"
```

Comment:
while the `v1alpha1` uses `data`, see: https://github.com/knative/eventing/blob/master/pkg/apis/sources/v1alpha1/ping_types.go#L77

the `v1alpah2` will use `jsonData`, see: https://github.com/knative/eventing/blob/master/pkg/apis/sources/v1alpha2/ping_types.go#L68

Reply:
@matzew Thanks ,  Currently based on CRD response `v1alpha1`  is picked as has both `served` and `storage` true so having `data`. Once it's `v1alpha2` we need to update `jsonData ` . 

Do you see that `served` and `storage` to be true  with 1.7.1 of serverless for 4.5 ??